pkg/tiller/kind_sorter.go

```diff
@@ -123,5 +123,9 @@ func (k *kindSorter) Less(i, j int) bool {
 	if !ok {
 		return true
 	}
+	if first == second {
+		// same kind so sub sort alphanumeric
+		return a.name < b.name
+	}
```

Comment:
I will do it now (but push later) because we also need the "template plugin in core" to sort exactly the same way so it'd be great to get it finalized. Thanks for the great reviews sir.